src/js/components/TextInput/index.d.ts

```diff
@@ -12,8 +12,8 @@ export interface TextInputProps {
   messages?: {enterSelect?: string,suggestionsCount?: string,suggestionsExist?: string,suggestionIsOpen?: string};
   name?: string;
   onSelect?: ((x: { target: React.RefObject<HTMLElement>['current'], suggestion: any }) => void);
-  onSuggestionsOpen?: ((...args: any[]) => any);
-  onSuggestionsClose?: ((...args: any[]) => any);
+  onSuggestionsOpen?: (() => void);
+  onSuggestionsClose?: (() => void);
```

Comment:
@oorestisime I need another pair of eyes to make sure this change is clear.

Reply:
looks good to me. we call it without any args and we don't treat any response so all good here